test-plone-5.1.x.cfg

```diff
@@ -0,0 +1,9 @@
+[buildout]
+extends =
+    https://raw.github.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.inflator
+
+[versions]
+plone.app.multilingual = 5.3.2
```

Comment:
Should we move that to https://github.com/4teamwork/ftw-buildouts/blob/master/test-versions-plone-5.cfg ?

Reply:
Moved to https://github.com/4teamwork/ftw-buildouts/pull/148 but I let this specific version pinning in here

Reply:
👍 merged